blog.html

```diff
@@ -4,59 +4,70 @@
-        </div>
-      <div class="centerfooter align-left clearfix:after">
-        <h3>Find us here!</h3>
-        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3102.822002430413!2d-77.34076030000001!3d38.9508946!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89b6482e2c339799%3A0xcf727a5bdcbdbc6c!2s1886+Metro+Center+Dr+%23620%2C+Reston%2C+VA+20190!5e0!3m2!1sen!2sus!4v1438570536650" width="400" height="300" allowfullscreen></iframe>
-      </div>
-      <div class="rightfooter align-right">
-        <h3>Disclaimer</h3>
-        <p>It should not be assumed that recommendations made in the future will be profitable or will equal the performance of the securities in this list. All performance calculations prior to 2013 have been done by the research department and have not been audited or verified by PWA or any outside 3rd party. The performance was calculated using live models and actual trades but not through PWA's platform. Past performance may not be indicative of future results and the performance of a specific individual client account may vary substantially from the composite performance results. Therefore, no current or prospective client should assume that future performance will be profitable, or equal either the Preston Wealth Advisors’ composite performance results reflected above, or the performance results for any of the comparative index benchmarks provided.</p>
+    </main>
 
-        <p id="show-this-on-click" class="hide">Preston Wealth Advisors (“PWA”) is an SEC registered investment adviser located in Reston, VA. PWA is registered in the state of Virginia and in compliance with the current registration requirements of the states in which PWA maintains clients. PWA may only transact business in those states in which it is registered, or qualifies for an exemption or exclusion from registration requirements.<br>
+    <section class="section-blog">
 
-        PWA’s web site is limited to the dissemination of general information pertaining to its advisory services, together with access to additional investment-related information, publications, and links. Accordingly, the publication of PWA’s web site on the Internet should not be construed by any consumer and/or prospective client as PWA’s solicitation to effect, or attempt to effect transactions in securities, or the rendering of personalized investment advice for compensation, over the Internet. Any subsequent, direct communication by PWA with a prospective client shall be conducted by a representative that is either registered or qualifies for an exemption or exclusion from registration in the state where the prospective client resides.</p>
-        <p class="readmore align-right"><a href="#">Read more &gt;</a></p>
-				<p class="readless align-right hide"><a href="#">Read less &gt;</a></p>
+    </section>
 
-      </div>
-        <p>©2015 Preston Wealth Advisors</p>
+    <footer class="footer-home">
+      <div class="margin">
```

Comment:
When creating a class margin, make sure you are explicit. Is it top margin, bottom, left or right? What's the sizing? 20px? For reusability and readability be detailed.